libtsuba/src/RDGPartHeader.h

```diff
@@ -472,6 +498,29 @@ class KATANA_EXPORT RDGPartHeader {
     return DoSelectProperties(storage_info);
   }
 
+  // Some property files are split up, so we make sure to keep track of all offset files
+  static katana::Result<std::vector<std::string>> GetOffsetFiles(
+      const std::vector<PropStorageInfo>& storage_info,
+      std::string prop_dir_name) {
+    std::vector<std::string> prop_offset_files;
+    auto reader = KATANA_CHECKED(tsuba::ParquetReader::Make());
+    for (const auto& prop : storage_info) {
+      auto full_path = katana::Uri::JoinPath(prop_dir_name, prop.path());
+      auto uri_path = KATANA_CHECKED(katana::Uri::Make(full_path));
+      auto num_files = KATANA_CHECKED(reader->NumFiles(uri_path));
+      // num_files can be 1, which means there are no split property files
+      if (num_files == 1) {
+        continue;
+      }
+      for (uint64_t i = 0; i < num_files; i++) {
+        auto file_name = fmt::format("{}.part_{:09}", prop.path(), i);
```

Comment:
I do wish we can centralize all RDG naming logic to one area, rather than having individual` fmt::format` calls.